Source/Custom Device Embedded Data Logger.xml

```diff
@@ -87,6 +87,22 @@
 				<Path>Custom Devices\National Instruments\Embedded Data Logger\Windows\Embedded Data Logger Configuration.llb\ActionVIOnLoad.vi</Path>
 			</ActionVIOnLoad>
 		</Page>
+		<Page>
+			<Name>
+				<eng>Commands Group Page</eng>
+				<loc>Commands Group Page</loc>
+			</Name>
+			<DeleteProtection>true</DeleteProtection>
```

Comment:
It's interesting to note that these were not previously delete-protected. They likely should have been, as the custom device doesn't function without them.